team/team.json

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
Is this file formatted? Using ESlint.

Reply:
No sir. I just used prettier plugin in vs code.